src/abi/tests.rs

```diff
@@ -246,7 +268,7 @@ fn instructions_and_types() {
     // sum function
     assert_eq!(idl.instructions[3].name, "sum");
     assert!(idl.instructions[3].docs.is_none());
-    assert!(idl.instructions[3].accounts.is_empty());
+    assert_eq!(idl.instructions[3].accounts.len(), 1);
```

Comment:
Instead of verifying the length, we should check if the right account is in the `accounts` vector.

Reply:
Fixed